tests/e2e/tests/pilot/ingressgateway_test.go

```diff
@@ -153,13 +153,17 @@ func TestGateway_TCPIngress(t *testing.T) {
 
 	for cluster := range tc.Kube.Clusters {
 		runRetriableTest(t, "TCPIngressGateway", defaultRetryBudget, func() error {
-			reqURL := fmt.Sprintf("http://%s.%s:31400/c", ingressGatewayServiceName, istioNamespace)
-			resp := ClientRequest(cluster, "t", reqURL, 100, "--key Host --val uk.bookinfo.com")
 			count := make(map[string]int)
-			for _, elt := range resp.Version {
-				count[elt]++
+			reqURL := fmt.Sprintf("http://%s.%s:31400/c", ingressGatewayServiceName, istioNamespace)
+			// We send request incrementally to avoid overloading sidecar/gateway all at once.
+			for i := 0; i < 10; i++ {
+				resp := ClientRequest(cluster, "t", reqURL, 10, "--key Host --val uk.bookinfo.com")
+				for _, elt := range resp.Version {
+					count[elt]++
+				}
+				log.Infof("request counts %+v", count)
+				time.Sleep(3 * time.Second)
```

Comment:
May try to reduce this. Ok to do in another PR

Reply:
sure.